lib/shell.js

```diff
@@ -5,11 +5,13 @@ const readline = require('readline');
 const RESTClient = require('./RESTClient');
 const host = process.argv[2] || 'localhost';
 const client = new RESTClient([host]);
-const Logger = require('werelogs').Logger;
-const logging = new Logger('shell', {
+const werelogs = require('werelogs');
+
+werelogs.configure({
     level: 'info',
     dump: 'error',
 });
+const logging = new werelogs.Logger('shell');
```

Comment:
Renamed into `BucketClientShell` for consistency with REST Client, and show that it's the module handling the cli commands.